.pre-commit-config.yaml

```
@@ -38,6 +38,12 @@ repos:
         additional_dependencies: [tomli]
         #args: ["--write-changes"] # uncomment if you want to get automatic fixing
 
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.11.11
```

this can or should be
```suggestion
    rev: v0.11.13
```